::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 13 May 2016 23:06:28 +0200
Subject: [PATCH] Add cobbler-grub-aarch64 to cobbler dependencies.

The aarch64 grub EFI loader, packaged as an RPM and previously
installed in the nailgun docker container in /var/lib/cobbler,
is now added as a cobbler dependency in puppet and installed
on the Fuel master node.

[Alexandru.Avadanii@enea.com]
Reworked based on Florin's work for the docker container.
Rebased for Fuel Newton.

Signed-off-by: Florin Dumitrascu <florin.dumitrascu@enea.com>
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 deployment/puppet/cobbler/manifests/packages.pp | 2 +-
 1 file changed, 1 insertions(+), 1 deletions(-)

diff --git a/deployment/puppet/cobbler/manifests/packages.pp b/deployment/puppet/cobbler/manifests/packages.pp
index 267c18c..9782c4c 100644
--- a/deployment/puppet/cobbler/manifests/packages.pp
+++ b/deployment/puppet/cobbler/manifests/packages.pp
@@ -23,5 +23,5 @@ class cobbler::packages {
       $django_package              = 'python-django'
       $openssh_package             = 'openssh-clients'
       $pexpect_package             = 'pexpect'
-      $cobbler_additional_packages = ['xinetd', 'tftp-server', 'syslinux', 'wget', 'python-ipaddr','fence-agents-all', 'bind-utils']
+      $cobbler_additional_packages = ['xinetd', 'tftp-server', 'syslinux', 'wget', 'python-ipaddr','fence-agents-all', 'bind-utils', 'cobbler-grub-aarch64']
     }
